flake.nix

```diff
@@ -1,21 +1,25 @@
 {
   description = "nixos-generators - one config, multiple formats";
 
+  # Lib dependency
+  inputs. = "github:divnix/nixpkgs.lib";
```

Comment:
Don't you miss that?
```suggestion
  inputs.nixlib.url = "github:divnix/nixpkgs.lib";
```

Reply:
ooops :smile: -- four eyes better than two. Thanks!